test/mpi/basictypetest.txt

```diff
@@ -43,5 +43,4 @@ rma/lockall_dt_flushall.c::1 8 64 512 32768 262144:timeLimit=1800:4
 rma/lockall_dt_flushlocal.c::1 8 64 512 32768 262144:timeLimit=1800:4
 rma/lockall_dt_flushlocalall.c::1 8 64 512 32768 262144:timeLimit=1800:4
 rma/putfence1.c::1 8 64 512 32768 262144::2
-rma/putfence1.c:LARGE_CNT:16000000:timeLimit=1800:2
```

Comment:
@minsii this is actually incorrect, what is not used is the `LARGE_CNT` macro not the line. I will fix this if you agree with other changes.

Reply:
Except the comment about the warning-fix commits, the other changes look good to me.